### PR TITLE
Return jobs_running from shutdown function

### PR DESCRIPTION
**Pull request by cstyan**
Return the number of jobs still running on a tunnel when calling the shutdown function.

cc @saucelabs/scdev

### CODE REVIEW

File: rest.go [hunk 1]
```diff
@@ -257,14 +257,14 @@ func (c *Client) Find(name string, domains []string) (
...
+func (c *Client) Shutdown(id string, response interface{}) error {
+	return c.shutdown("%s/%s/tunnels/%s", id, &response)
 }
 
-func (c *Client) shutdown(urlFmt, id string) error {
+func (c *Client) shutdown(urlFmt, id string, response interface{}) error {
 	var url = fmt.Sprintf(urlFmt, c.BaseURL, c.Username, id)
 
-	return c.executeRequest("DELETE", url, nil, nil)
+	return c.executeRequest("DELETE", url, nil, &response)
```
Comment (henryprecheur) [inline rest.go]: I don't think it's the best interface to do this. I'd rather return the number of running jobs from shutdown directly:

```go
func (c *Client) shutdown(urlFmt, id string) (jobsRunning uint, err error) {
	var url = fmt.Sprintf(urlFmt, c.BaseURL, c.Username, id)

	var response = struct {
		JobsRunning uint `json:"jobs_running"`
	}
	err = c.executeRequest("DELETE", url, nil, &response)
	jobsRunning = response.JobsRunning

	return
}
```

I'd like to try to hide all the JSON structures inside the REST lib.

Comment (cstyan) [inline rest.go]: I agree, I didn't like passing the interface in every function call. Any particular reason for using `uint` rather than just `int`?

Comment (ldx) [inline rest.go]: Agree!

**Comment by henryprecheur** [comment]
Run `gofmt -w .` on it to reformat properly and you're :ship:

**Comment by ldx** [comment]
:+1:

**Comment by cstyan** [comment]
`go fmt` changes indentation from tabs to spaces, weird

**Pull request merged**